extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java

```diff
@@ -105,6 +104,16 @@ private String findInitialCommentsToken(final JsonObject nextResponse)
         return token;
     }
 
+    @Nonnull
+    private JsonArray getJsonContents(final JsonObject nextResponse) {
+        try {
+            return JsonUtils.getArray(nextResponse,
+                    "contents.twoColumnWatchNextResults.results.results.contents");
+        } catch (final ParsingException pe) {
```

Comment:
You don't need to use `pe`, you could just `e` for the exception in this context:

```suggestion
        } catch (final ParsingException e) {
```